test/functional/wallet_simulaterawtx.py

```diff
@@ -46,41 +46,87 @@ def run_test(self):
 
         # Add address1 as watch-only to w2
         w2.importpubkey(pubkey=w1.getaddressinfo(address1)["pubkey"])
-
```

Comment:
In bf9deb407a8d73ac3b645b34c9bd6d28729a809c "rpc/wallet: Add details and duplicate section for simulaterawtransaction"

nit: Unrelated whitespace change